packages/microsite/src/cli/microsite-dev.tsx

```diff
@@ -112,14 +112,14 @@ const renderPage = async (page: string, initialProps?: any) => {
     const match =
       paths &&
       paths.find(
-        (ctx) => ctx.path === pathname || ctx.path === `${pathname}/index`
+        (ctx) => ctx.path === pathname || ctx.path === `${pathname}/index` || ctx.path === absoluteUrl
```

Comment:
It might be that the `absoluteUrl` should be normalized in some way like the `ctx.path` (honestly I haven't looked at the normalization logic in the `generateStaticPropsContext` yet)

Reply:
Yep, looks like this fixes it!